controller/endpointcontroller.php

```diff
@@ -70,7 +70,7 @@ public function get() {
 		// We stop polling for them.
 		if (!$this->manager->hasNotifiers()) {
 			$response = new Response();
-			$response->setStatus(Http::STATUS_NOT_FOUND);
+			$response->setStatus(Http::STATUS_NO_CONTENT);
 			return $response;
```

Comment:
from a pure rest api pov returning a simple empty array would be valid as well - no need for any other error code